### PR TITLE
Display error for radio lists if provided

### PR DESCRIPTION
**Pull request by ztolley**
error
The radio control accepts a error parameter, which is an error by
convention. The control will then conditionally set the has-error class
on the wrapper and display the first error below the field.

### CODE REVIEW

File: app/helpers/barnardos/action_view/form_helpers.rb [hunk 1]
```diff
@@ -90,6 +91,12 @@ def radio_group_vertical(name, legend, selection_list, value = nil, error: nil,
                 end
               )
             end
+            concat(
+              content_tag(:div,
+                          "#{name.to_s.humanize} #{error[0]}",
+                          class: 'radio-group__error'
+              )
+            ) if error.present?
```
Comment (rgarner) [inline app/helpers/barnardos/action_view/form_helpers.rb]: If testing for `nil`, can just say `if error` here (any truthy value will do) – `present` is for when you need to test for nil or empty, in the case of a `String` that might be `""` or an array that might be `[]`

File: app/helpers/barnardos/action_view/form_helpers.rb [hunk 2]
```diff
@@ -90,6 +91,14 @@ def radio_group_vertical(name, legend, selection_list, value = nil, error: nil,
                 end
               )
             end
+
+            concat(
+              content_tag(:div,
+                          "#{name.to_s.humanize} #{error[0]}",
```
Comment (rgarner) [inline app/helpers/barnardos/action_view/form_helpers.rb]: Should this just be a string at this stage? We have the singular `error` but we're passing an array.

Comment (ztolley) [inline app/helpers/barnardos/action_view/form_helpers.rb]: see checkbox comment

**Comment by rgarner** [comment]
@ztolley could you rebase? Thanks
